presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -737,6 +741,27 @@ public String getField()
             return nullOrdering;
         }
 
+        public SortOrder getSortOrder()
+        {
+            checkState(nullOrdering != UNDEFINED, "nullOrdering is undefined");
+            if (ordering == ASCENDING) {
```

Comment:
it looks like it is copied from somewhere... maybe you could extract some utility and reuse it here?